src/core/atomicdex/utilities/cpprestsdk.utilities.cpp

```diff
@@ -57,7 +57,7 @@ handle_exception_pplx_task(pplx::task<void> previous_task)
 
 pplx::task<std::filesystem::path> download_file(const t_http_client_ptr& client, const std::string& url, const std::filesystem::path& output_location)
 {
-    SPDLOG_DEBUG("Start downloading {} to {}", url, output_location);
+    SPDLOG_DEBUG("Start downloading {} to {}", url, output_location.string());
```

Comment:
This will not compile also on windows because `string()` expand to `wstring` for `filesystem` on windows, you need to use `.c_str()`